src/components/views/RealTimeChart.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Plot from 'react-plotly.js';
+
+import Error from '../views/Error';
+import Loading from '../views/Loading';
+
+import './css/RealTimeChart.css';
```

Comment:
```
Failed to compile.

./src/components/views/RealTimeChart.js
Module not found: Can't resolve './css/RealTimeChart.css' in '/Users/john-moz/devel/projects/firefox-test-tube/src/components/views'
```

Reply:
I don't get that locally and deployed fine on staging.

Reply:
Ah, I forgot we were using Stylus for TT. I'll need to restart the server. You're good.